.github/checklist.yml

```diff
@@ -0,0 +1,10 @@
+paths:
+  "Cargo.lock":
+    - Did you review changed dependencies for security concerns?
+    - Has at least one other team member approved the dependency changes?
+  "rfcs/**":
+    - Have at least 3 team members approved this RFC?
+  "src/**":
+    - Did you add adequate behavior tests (`tests/behavior`)? Specifically, nested fields support.
```

Comment:
I think the question about the behavior tests should relate only to `src/transforms/**`.